tensorflow/core/kernels/stateful_random_ops.cc

```diff
@@ -16,9 +16,9 @@ limitations under the License.
 #define EIGEN_USE_THREADS
 
 #include "tensorflow/core/framework/tensor_util.h"
+#include "tensorflow/core/framework/training_op_helpers.h"
```

Comment:
`tensorflow/core/kernels:stateful_random_ops does not depend on a module exporting 'third_party/tensorflow/core/framework/training_op_helpers.h'`